cmd/nerdctl/volume_prune_linux_test.go

```diff
@@ -17,30 +17,137 @@
 package main
 
 import (
-	"fmt"
+	"strings"
 	"testing"
 
+	"gotest.tools/v3/assert"
+	"gotest.tools/v3/icmd"
+
 	"github.com/containerd/nerdctl/v2/pkg/testutil"
 )
 
 func TestVolumePrune(t *testing.T) {
-	base := testutil.NewBase(t)
-	tID := testutil.Identifier(t)
-	base.Cmd("volume", "prune", "-a", "-f").Run()
+	// Volume pruning cannot be parallelized for Docker, since we need namespaces to do that in a way that does interact with other tests
+	if testutil.GetTarget() != testutil.Docker {
+		t.Parallel()
+	}
+
+	// FIXME: for an unknown reason, when testing ipv6, calling NewBaseWithNamespace per sub-test, in the tearDown/tearUp methods
```

Comment:
No idea.........
I guess it is another thing to add to the list for whomever is going to cleanup our testing helpers eventually.